### DISCUSSION TITLE
Online Flow Bench Resources

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#50] [Flow Bench Design]
Here's a list of interesting or helpful websites, articles, blogs, vendors etc.

If you find something of use, please share it in this thread :)

I'll start off with our very own DIY Flow Bench Facebook page. This is a fairly active page with a reasonably large membership (3600 at the time of this post) and regular discussions covering a variety of topics. Whilst it was initially created to support the DIYFB project, it has grown into a diverse group that includes all kinds of benches and projects which is really cool. The organic growth of the Facebook page is the reason that the DIYFB project has now been moved here to its own dedicated website.

www.facebook.com/groups/diyflowbench

**Comment by DeeEmm** [comment]
The Dtec Flowbench Design Guide page has a pretty good basic design for a traditional orifice style flow bench. There's lots of great info there. Well worth taking the time to read through.

[dtec.net.au/Flowbench Design Guide.htm](http://dtec.net.au/Flowbench%20Design%20Guide.htm)

**Comment by DeeEmm** [comment]
This Musclecar DIY article is an excerpt from the David Vizard 'how to port and flow cylinder heads' book, which in my opinion is a must read for anyone considering making their own flow bench. The article appears to be labelled part 3, however I have been unable to find Parts 1 & 2. It's basically a snippet from the book that's been published to generate book sales.

[https://www.musclecardiy.com/cylinder-heads/build-flow-bench-port-flow-testing-cylinder-heads-part-3/](https://www.musclecardiy.com/cylinder-heads/build-flow-bench-port-flow-testing-cylinder-heads-part-3/)

**Comment by DeeEmm** [comment]
Performance Trends EZ Flow System

This is a vendor page for the 'EZ Flow System'. It has some useful design layouts of a basic design using an orifice style setup that could easily be adapted to a MAF style bench.

[https://performancetrends.com/ez_flow_system.htm](https://performancetrends.com/ez_flow_system.htm)

The plans can be viewed at the following link

[https://performancetrends.com/PDFs/EZ%20Flow%20w%20Black%20Box.pdf](https://performancetrends.com/PDFs/EZ%20Flow%20w%20Black%20Box.pdf)

**Comment by DeeEmm** [comment]
An interesting 'Instructable' put together by a uni team who built a basic flow bench for a uni project. Whilst it's not the best example, it does have some good info. They also appear to use the MAF from a Volvo S80 which may be a good candidate for use with a DIY bench as they say that the datasheet is available to download.

[www.instructables.com/id/Building-a-Flow-Bench/](www.instructables.com/id/Building-a-Flow-Bench/)

They also link to the actual project data they created which includes further technical information and conclusions relating to their project.

[https://blogs.brown.edu/engn1860rd/spring-2019-projects/group-1/group-1-constructing-and-testing-a-low-cost-flow-bench/](https://blogs.brown.edu/engn1860rd/spring-2019-projects/group-1/group-1-constructing-and-testing-a-low-cost-flow-bench/)

**Comment by DeeEmm** [comment]
The IVAN2104 youtube channel has a brief rundown on a basic floating reference manometer build.

https://youtu.be/cg77Uz31EX8

**Comment by jwestes** [comment]
This one results in a 404 error.  That is a great book though.  I've read it a couple times.

Jason Estes
***@***.***
________________________________

**Reply by DeeEmm** [reply]
> This one results in a 404 error. That is a great book though. I've read it a couple times. Jason Estes ***@***.***

Thanks for the heads up. I've updated the link in the post. It must have got garbled when I cut and pasted it across from the website forums. Just trying to retain some of the useful info from the forums before I deactivate them.
